server/handlers/queue.go

```diff
@@ -151,7 +179,14 @@ func PublishBulk(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, gin.H{"error": "tries shouldn't be zero"})
 		return
 	}
-
+	attributes := make(map[string]string)
+	for key, vals := range c.Request.Header {
+		if !strings.HasPrefix(key, jobAttributeHeaderPrefix) || len(vals) == 0 {
+			continue
+		}
+		field := strings.TrimPrefix(key, jobAttributeHeaderPrefix)
+		attributes[field] = vals[0]
+	}
```

Comment:
```suggestion
	attributes := make(map[string]string)
	for key, vals := range c.Request.Header {
		if !strings.HasPrefix(key, jobAttributeHeaderPrefix) || len(vals) == 0 {
			continue
		}
		field := strings.TrimPrefix(key, jobAttributeHeaderPrefix)
		attributes[field] = vals[0]
	}
```
Move this part into a function